docs/concepts/use-cases.md

```diff
@@ -137,4 +139,4 @@ Vulnerability Management Integration Example: [Tenable.io](https://xsoar.pan.dev
 **Top Use Cases:**
 - Create/update/delete users.
 - Block user/Force change password
-- Managing access to resources and applications.
\ No newline at end of file
+- Managing access to resources and applications.
```

Comment:
Do you want to add a comment about policies and roles? We have CRUD functions on both of these in AWS-IAM and will have the same in GCP-IAM.

Reply:
But not in Okta-IAM, GitHub-IAM, Workday-IAM.
added partially, per the functionalities I saw in the majority of them.